xivo_fetchfw/package.py

```diff
@@ -596,7 +592,7 @@ def filter_fun(dep_pkg_id):
         return installed_pkgs
 
 
-class UpgraderController(object):
+class UpgraderController:
     # TODO add comments
```

Comment:
This TODO could have been removed. That is not necessary in this case :sweat_smile: